frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/utils/treatment-progress-to-german-name.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { TreatmentProgress } from 'digital-fuesim-manv-shared';
+const treatmentProgressToGermanNameDictionary: {
+    [Key in TreatmentProgress]: string;
+} = {
+    counted: 'Vorsichten',
+    secured: 'Erstversorgung sichergestellt',
+    triaged: 'Behandeln, Personal fehlt',
+    unknown: 'Erkunden',
```

Comment:
It would certainly be helpful to add a treatment progress for 'not started' (e.g if there is no personnel) because `Erkunden` is misleading in that case.

Reply:
I also think that would be nice but adding more of those would involve altering the behaviour, which is not part of this issue